### PR TITLE
Add MOON token

### PR DESCRIPTION
**Pull request by axldefi**

### CODE REVIEW

**Review by ecdsafu — CHANGES_REQUESTED** [review]
For this token icon we'll need a png in order to keep the size down. Could you please use this instead? Just exported it to have a file size of <32kb. You're welcome to provide your own if you have one that's better.

moon

**Comment by axldefi** [comment]
> For this token icon we'll need a png in order to keep the size down. Could you please use this instead? Just exported it to have a file size of <32kb. You're welcome to provide your own if you have one that's better.
> 
> moon

I'm perfectly good with that version of the token, thank you!

**Comment by canhtrinh** [comment]
> > For this token icon we'll need a png in order to keep the size down. Could you please use this instead? Just exported it to have a file size of <32kb. You're welcome to provide your own if you have one that's better.
> > moon
> 
> I'm perfectly good with that version of the token, thank you!

we'll need svgs. @axldefi any way you can take that png and convert that to a low(er)-fidelity svg?

**Comment by axldefi** [comment]
> > > For this token icon we'll need a png in order to keep the size down. Could you please use this instead? Just exported it to have a file size of <32kb. You're welcome to provide your own if you have one that's better.
> > > moon
> > 
> > 
> > I'm perfectly good with that version of the token, thank you!
> 

> we'll need svgs. @axldefi any way you can take that png and convert that to a low(er)-fidelity svg?

will this work? 

moon (2)

**Comment by ecdsafu** [comment]
> > > > For this token icon we'll need a png in order to keep the size down. Could you please use this instead? Just exported it to have a file size of <32kb. You're welcome to provide your own if you have one that's better.
> > > > moon
> > > 
> > > 
> > > I'm perfectly good with that version of the token, thank you!
> 
> > we'll need svgs. @axldefi any way you can take that png and convert that to a low(er)-fidelity svg?
> 
> will this work?
> 
> moon (2)

Yep it's a bit big, but that'll be good enough

File: registry/mainnet/interchain/squid.tokenlist.json [hunk 1]
```diff
@@ -415,6 +415,68 @@
           "tokenManagerType": "mintBurn"
         }
       ]
-    }
+    },
+    "0xf5ddb0c04985ee286ca7930c8dc38bcb5e71f123027cb8886b0dd834578120bd": {
```
Comment (canhtrinh) [inline registry/mainnet/interchain/squid.tokenlist.json:419]:
```suggestion
    { "0xf5ddb0c04985ee286ca7930c8dc38bcb5e71f123027cb8886b0dd834578120bd": {
```

**Review by canhtrinh — APPROVED** [review]

**Pull request merged**